docker/Dockerfile.xenial_debug

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial
+# Base image for debug builds.
+# Built manually uploaded as "istionightly/base_debug"
+
+# Do not add more stuff to this list that isn't small or critically useful.
+# If you occasionally need something on the container do
+# sudo apt-get update && apt-get whichever
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+      curl \
```

Comment:
can we add fortio :-)

Reply:
I know you like fortio  - but please don't add more stuff here. There is a docker image for it.

Reply:
it’s 2mb and is useful (to me and orlthers for debugging)